src/api/getProduct.ts

```diff
@@ -0,0 +1,12 @@
+import { API_ENDPOINTS } from "../config/api";
+import { Product } from "./types";
+import { makeGetRequest } from "../utils/axios_get";
+
+//https://api.escuelajs.co/api/v1/products/4
+//надо приклетить id к url
+const getProduct = async (id: number ) => {
```

Comment:
```suggestion
const getProduct = async (id: number ): Promise<Product | null> => {
```
или
`const getProduct = async (id: number ): Promise<BaseResponse<Product>> => {`